docs/endpoint.md

```diff
@@ -106,37 +106,10 @@ $ sudo apt-get install -y   \
     zlib1g-dev
 ```
 
-Create a workspace
-
-```shell
-$ mkdir -p workspace
-$ cd workspace
-```
-
-Clone a specific version of `legato`. The `19.07.0` is the preferred stable version.
-
-```shell
-$ repo init -u git://github.com/legatoproject/manifest -m legato/releases/19.07.0.xml # specific legato 19.07.0 version
-$ repo sync
-```
-
-Build legato as native target
-
-```shell
-$ cd legato
-$ make localhost
-```
-
-Checkout the shell to legato shell
+Use GNU Make to build endpoint application at the root directory of tangle-accelerator.
 
 ```shell
-$ source framework/tools/scripts/configlegatoenv
-```
-
-Finally, use GNU Make to build endpoint application at the root directory of tangle-accelerator.
-
-```shell
-$ cd tangle-accelerator
+$ ./prebuild-endpoint.sh # run legato prebuild script
```

Comment:
It is confusing to run `prebuilt-endpoint.sh`. Move into `legato` target or something similar.